CHANGES/7401.misc

```diff
@@ -0,0 +1 @@
+Syncing defaults to policy 'on_demand' now.
```

Comment:
> Change the default download policy to 'on_demand', as bulk syncs of PyPI would use several terabytes of disk space.

Reply:
You don't have to use that ^ but a little more detail would be helpful I think.

Also, we should fix the http responses on this page, and flesh out the documentation to cover the following:

* An example of a full-pypi mirror and a note about the changed default from what is typical with Pulp, and why (terabytes of disk needed)
* Fix the includes / excludes specifiers since I think they are still using the "old" format
* Note that you do a full-pypi mirror by simply not using the "includes" field

Reply:
Not sure if this got hidden by the "outdated" but it's still applicable.

Reply:
@gerrod3 ^

Reply:
Thanks!